FontAwesome/Enum.swift

```diff
@@ -2798,6 +2798,7 @@ public enum FontAwesome: String {
             case .youtube: return "\u{f167}"
             case .youtubeSquare: return "\u{f431}"
             case .zhihu: return "\u{f63f}"        
+            default: return ""
```

Comment:
This should be done in `codegen.swift` and not directly in the `Enum.swift` file

Reply:
How do I run codegen.swift? It is failing at
```
guard let json = FileManager.default.contents(atPath: "FortAwesome/Font-Awesome/metadata/icons.json") else {
    fatalError("Could not find JSON metadata file")
}
```
I don't see the file icons.json inside the git repo.

Reply:
The file is in a submodule in `FortAwesome/FontAwesome`

Reply:
I have updated codegen.swift, Enum.swift seems to be in the repo as well so I left the changes there.